script/deprecated-imports.csv

```diff
@@ -8,8 +8,10 @@ src/Nri/Ui/Page/V3.elm,Nri.Ui.Text,2
 src/Nri/Ui/SlideModal/V2.elm,Nri.Ui.Button,8
 src/Nri/Ui/SlideModal/V2.elm,Nri.Ui.Icon,3
 src/Nri/Ui/SlideModal/V2.elm,Nri.Ui.Text,2
+src/Nri/Ui/TextInput/V6.elm,Nri.Ui.InputStyles,2

```

Comment:
Should we update these..?

Reply:
I mean, yeahhhhhhhhhhhh but also

![](https://media1.tenor.com/images/98d6d61e7aebac2dddfb00d1d1750501/tenor.gif?itemid=17563254)

If you really think we should, we can, but it seems to require bumping both versions to their next `Vwhatever` and this is already going kind of off our critical path for TDA right now.

Reply:
Oh -- i'm surprised that's not a patch change! happy to wait then